### ISSUE TITLE
Invoke-GlobalMailSearch is searching/exporting every email twice

### ISSUE THREAD

**Issue by bobosnubs** [dafthack/MailSniper#53]
Invoke-GlobalMailSearch -ImpersonationAccount "user@domain.com" -ExchHostname "outlook.office365.com" -AdminUserName "admin@domain.com" -AdminPassword "password" -Terms * -MailsPerUser 500 -OutputCsv searchedmatches.csv

^^^ _Here is the code that I am running_ ^^^

Specifically, my issue is that Invoke-GlobalMailSearch is searching/exporting every single email twice. When I run _(Get-Mailbox).count_ it returns _37_, aka 37 users. However when I run the above code it will give a display of _[1/76]_ mailboxes, _[2/76]_, _[3/76]_... _[76/76]_. 

Basically how, if I only have 37 users, does MailSniper search 76 total mailboxes? And how can I solve this?

Huge thanks for any responses!

**Comment by dafthack** [comment]
That issue is likely due to there being multiple email addresses associated with each user. Someone else reported a similar issue before. 

Try modifying the following line in the script by replacing ‘-ExpandProperty EmailAddresses’ with ‘-ExpandProperty primarySmtpAddress' and run it again. 

`$SMTPAddresses = Get-Mailbox -ResultSize unlimited | Select Name -ExpandProperty EmailAddresses`

Let me know if that helps at all!

**Comment by bobosnubs** [comment]
Yay! It seems to be working now, thank you very much Beau!

**Comment by dafthack** [comment]
Awesome. I'll test out modifying that in the master script and push it some time soon.
